app/assets/stylesheets/style.css

```diff
@@ -75,6 +75,31 @@ h6.dropdown-header {
   border: none;
 }
 
+
+.note-show h1, #main-content h1 {
+  font-size: 1.95rem;
+}
+
+
+@media (max-width:400px) {
+  .note-show h1, #main-content h1 {
+    font-size: 1.6rem;
```

Comment:
@illyShelly Could we make this `2rem`?

Reply:
Also we can remove the #main-content selectors from these styles. I looked at some wiki pages and the fonts-sizes are responsive.

https://user-images.githubusercontent.com/63427719/167269409-cfd0a27c-6eca-4eed-ab67-2bbb137b3538.mov

Reply:
Also, could we use this media query for just `h1` and `h2` tags? Thanks!!

Reply:
Hello @TildaDares, okay.